dev-packages/browser-integration-tests/suites/stacktraces/init.js

```diff
@@ -4,4 +4,10 @@ window.Sentry = Sentry;
 
 Sentry.init({
   dsn: 'https://public@dsn.ingest.sentry.io/1337',
+  transportOptions: {
+    fetchOptions: {
+      // See: https://github.com/microsoft/playwright/issues/34497
+      keepalive: false,
```

Comment:
Important note: This is apparently a bug in playwright/firefox 🤔 I opened an issue here: https://github.com/getsentry/sentry-javascript/pull/15164 but this seems to fix it. I tried to reproduce this in a regular FF instance, and it seems that Sentry data is correctly sent, but cannot be inspected. Possibly this interferes with PW abilities to intercept stuff too.